src/renderer/components/TimelineSpace/Contents/Mentions.vue

```diff
@@ -0,0 +1,236 @@
+          })
+      }
+    },
+    focusedId: function (newState, oldState) {
+      if (newState && this.heading) {
+        this.$store.commit('TimelineSpace/Contents/Mentions/changeHeading', false)
+      } else if (newState === null && !this.heading) {
+        this.$store.commit('TimelineSpace/Contents/Mentions/changeHeading', true)
+        this.$store.commit('TimelineSpace/Contents/Mentions/mergeMentions')
+      }
+    }
+  },
+  methods: {
+    onScroll (event) {
+      // for lazyLoading
+      if (((event.target.clientHeight + event.target.scrollTop) >= document.getElementById('mentions').clientHeight - 10) && !this.lazyloading) {
+        this.$store.dispatch('TimelineSpace/Contents/Mentions/lazyFetchMentions', this.mentions[this.mentions.length - 1])
+          .catch(() => {
+            this.$message({
+              message: this.$t('message.timeline_fetch_error'),
+              type: 'error'
+            })
+          })
+      }
+      // for unread control
+      if ((event.target.scrollTop > 10) && this.heading) {
+        this.$store.commit('TimelineSpace/Contents/Mentions/changeHeading', false)
+      } else if ((event.target.scrollTop <= 10) && !this.heading) {
+        this.$store.commit('TimelineSpace/Contents/Mentions/changeHeading', true)
+        this.$store.commit('TimelineSpace/Contents/Mentions/mergeMentions')
+      }
+    },
+    async reload () {
+      this.$store.commit('TimelineSpace/changeLoading', true)
+      try {
+        await this.reloadable()
+      } finally {
+        this.$store.commit('TimelineSpace/changeLoading', false)
+      }
+    },
+    updateToot (message) {
+      this.$store.commit('TimelineSpace/Contents/Mentions/updateToot', message)
+    },
+    upper () {
+      scrollTop(
+        document.getElementById('scrollable'),
+        0
+      )
+      this.focusedId = null
+    },
+    focusNext () {
+      const currentIndex = this.mentions.findIndex(toot => this.focusedId === toot.id)
+      if (currentIndex === -1) {
+        this.focusedId = this.mentions[0].id
+      } else if (currentIndex < this.mentions.length) {
+        this.focusedId = this.mentions[currentIndex + 1].id
+      }
+    },
+    focusPrev () {
+      const currentIndex = this.mentions.findIndex(toot => this.focusedId === toot.id)
+      if (currentIndex === 0) {
+        this.focusedId = null
+      } else if (currentIndex > 0) {
+        this.focusedId = this.mentions[currentIndex - 1].id
+      }
+    },
+    focusNotification (message) {
+      this.focusedId = message.id
+    },
+    focusSidebar () {
+      Event.$emit('focus-sidebar')
+    },
+    handleKey (event) {
+      switch (event.srcKey) {
+        case 'next':
+          this.focusedId = this.mentions[0].id
+          break
+      }
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+#mentions {
+  .unread {
+    position: fixed;
+    right: 24px;
+    top: 48px;
+    background-color: rgba(0, 0, 0, 0.7);
+    color: #ffffff;
```

Comment:
__[stylelint]__ Expected "#ffffff" to be "#fff" (color-hex-length) ([link](https://sider.review/gh/repos/124234495/pulls/849#issue-3838670))

<sub>posted by Sider</sub>